package.json

```diff
@@ -19,7 +19,8 @@
     "react-bootstrap": "1.0.0-beta.5",
     "react-dom": "16.8.4",
     "react-scripts": "2.1.3",
-    "truffle": "5.1.39",
+    "solc": "^0.8.0",
+    "truffle": "^5.5.32",
```

Comment:
please note that this may imapct other developer , please inform Yara and ensure she is able to update her code without problems